docs/source/intro/datasets.rst

```diff
@@ -126,27 +126,27 @@ Citation Managers
 
-+-----------------+---------------+----------------+--------------+--------------+
-|                 | **.ris**      | **.tsv**       | **.csv**     | **.xlsx**    |
-+-----------------+---------------+----------------+--------------+--------------+
-| **Endnote**     | Supported     | Not supported  |              |              |
-+-----------------+---------------+----------------+--------------+--------------+
-| **Excel**       |               |                | Supported\*  |  Supported   |
-|                 |               |                |              |              |
-+-----------------+---------------+----------------+--------------+--------------+
-| **Mendeley**    | Supported     |                |              |              |
-|                 |               |                |              |              |
-+-----------------+---------------+----------------+--------------+--------------+
-| **Refworks**    | Supported     | Not supported  |              |              |
-|                 |               |                |              |              |
-+-----------------+---------------+----------------+--------------+--------------+
-| **Zotero**      | Supported     |                | Supported    |              |
-|                 |               |                |              |              |
-+-----------------+---------------+----------------+--------------+--------------+
-
-- Supported: The data can be exported from the citation manager and imported in ASReview using this extension.
-- Not supported: The exported format can not be imported in ASReview.
-- (empty): The data cannot be exported from the citation manager in this format.
++-----------------+---------------+--------------+--------------+
+|                 | **.ris**      |  **.csv**    | **.xlsx**    |
++-----------------+---------------+--------------+--------------+
+| **Endnote**     | Supported     |              |              |
++-----------------+---------------+--------------+--------------+
+| **Excel**       |               |  Supported\* |  Supported   |
+|                 |               |              |              |
++-----------------+---------------+--------------+--------------+
+| **Mendeley**    | Supported     |              |              |
+|                 |               |              |              |
++-----------------+---------------+--------------+--------------+
+| **Refworks**    | Supported     |              |              |
+|                 |               |              |              |
++-----------------+---------------+--------------+--------------+
+| **Zotero**      | Supported     |  Supported   |              |
+|                 |               |              |              |
++-----------------+---------------+--------------+--------------+
+
+- Supported: The data can be exported from the citation manager and imported
+in ASReview.  
+- Empty: This format does not exist.
```

Comment:
Are you fine with changing this into `This export format is not available. `

Reply:
no, because then it seems it is not available in ASReview, but the formats simply do not exist in the citation managers

Reply:
This is the main reason why there was a Not Supported option. The confusing thing is, we support them all in this table ;) 

With the proposed changes, the empty option has different descriptions/meanings(?) across the page. I'm fine with changing the text somehow, but I prefer to have identical descriptions.

Reply:
I have added  the following to the tables:

-  ✅ = The data can be exported from the citation manager and imported in ASReview. 
-  N/A = This format does not exist.
-  X = not suported.